skills/docx/SKILL.md

```
@@ -354,73 +365,70 @@ Check that all changes appear correctly in the output.
+    ])
+    doc.save()
 ```
 
-If the edit appears in an unexpected location, reject it and retry with more context or different occurrence.
+If any hash is stale, the entire batch is rejected before any edits are applied.
 
 ### Workflow for Large Documents
 
-1. **Explore structure first** with python-docx:
+1. **List paragraphs** with hash-anchored references:
    ```python
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)